widget_scripts.md

```diff
@@ -1,3 +1,112 @@
 
 # Widget scripts
 
+### Widgets (HORUS ONLY) Scripts
+
```

Comment:
I would remove one of the heading, seems duplicated now.

Reply:
What is the prefered level? I just copied from another, but it seems way to low.

reduce the number of # on all titles??